proto/tendermint/types/types.proto

```diff
@@ -81,17 +81,26 @@ message Header {
   bytes proposer_address = 14;  // original proposer of the block
 }
 
-// Data contains the set of transactions included in the block
+// Data contains all the information needed for a consensus full node to
+// reconstruct an extended data square.
 message Data {
-  // Txs that will be applied by state @ block.Height+1.
-  // NOTE: not all txs here are valid.  We're just agreeing on the order first.
-  // This means that block.AppHash does not include these txs.
+  // Txs that will be applied to state in block.Height + 1 because deferred execution.
+  // This means that the block.AppHash of this block does not include these txs.
+  // NOTE: not all txs here are valid. We're just agreeing on the order first.
   repeated bytes txs = 1;
 
   // field number 2 is reserved for intermediate state roots
   // field number 3 was previously used for evidence
+
+  // Blobs are the data blobs included in the original data square.
   repeated Blob blobs       = 4 [(gogoproto.nullable) = false];
+
+  // SquareSize is the number of rows in the original data square.
```

Comment:
[optional]
this is definitely still correct, but perhaps we should also mention cols as well?